jazz_environments/components/crud/get.js

```diff
@@ -42,7 +42,7 @@ module.exports = (service, domain, environment_id, onComplete) => {
         };
     } else if (service !== undefined && domain !== undefined && environment_id === undefined) {
```

Comment:
If and else if are exact same conditions at line 33 and 43. Is this a bug in internal?

Reply:
If and else if conditions are not same. We are doing check for environment_id availability. So it is not a bug.